### PR TITLE
Release v0.36.0

### PR DESCRIPTION
**Pull request by cesnietor**
See changelog

### CODE REVIEW

**Review by pjuarezd — DISMISSED** [review]

**Review by cniackz — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,8 +2,36 @@
 
 # Changelog
 
+## Release v0.36.0
+
+Bug Fix:
+
+- Show LDAP Enabled value LDAP configuration
```
Comment (bexsoft) [inline CHANGELOG.md:9]:
```suggestion
- Show LDAP Enabled value LDAP configuration
- Updated Settings page components
```

**Review by bexsoft — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (cesnietor) [reply CHANGELOG.md:13]: somehow I missed it :D thanks

Comment (cesnietor) [reply CHANGELOG.md:13]: done, added as feature

**Review by cniackz — APPROVED** [review]
LGTM

**Review by pjuarezd — APPROVED** [review]

**Review by bexsoft — APPROVED** [review]

**Pull request merged**